config/config/sc-config.yaml

```diff
@@ -1061,6 +1061,26 @@ networkPolicies:
 #     ingress: {}
 #     egress: {}
 # ---
+  global:
+    apiserver:
+      ips: # usualy private ip of control-plane nodes
+        - set-me
+      port: 6443
+    scNodes:
+      ips: # ip of all nodes in the cluster for internal communication
+        - set-me
+    wcIngress:
+      ips: # ip(s) to wc loadbalancer if available, otherwise wc worker nodes
+        - set-me
+    objectStorage:
```

Comment:
This should be moved to common-config since we have velero writing directly to it.
So not really needed right now, but should be in the future

Reply:
Moved it now